src/20230816_AaveV3_Eth_ChaosLabsScopeAndCompensationAmendment/AaveV3_Ethereum_ChaosLabsScopeAndCompensationAmendment_20230816.t.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets, ICollector} from 'aave-address-book/AaveV2Ethereum.sol';
+import {IStreamable} from 'aave-address-book/AaveMisc.sol';
+import {ProtocolV3TestBase} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {AaveV3_Ethereum_ChaosLabsScopeAndCompensationAmendment_20230816} from './AaveV3_Ethereum_ChaosLabsScopeAndCompensationAmendment_20230816.sol';
+
+/**
+ * @dev Test for AaveV3_Ethereum_ChaosLabsScopeAndCompensationAmendment_20230816
+ * command: make test-contract filter=AaveV3_Ethereum_ChaosLabsScopeAndCompensationAmendment_20230816
+ */
+contract AaveV3_Ethereum_ChaosLabsScopeAndCompensationAmendment_20230816_Test is
+  ProtocolV3TestBase
+{
+  AaveV3_Ethereum_ChaosLabsScopeAndCompensationAmendment_20230816 internal proposal;
+
+  IERC20 public constant AUSDT = IERC20(AaveV2EthereumAssets.USDT_A_TOKEN);
+
+  ICollector public immutable AAVE_COLLECTOR = AaveV2Ethereum.COLLECTOR;
+  address public constant CHAOS_LABS_TREASURY = 0xbC540e0729B732fb14afA240aA5A047aE9ba7dF0;
+
+  IStreamable public immutable STREAMABLE_AAVE_COLLECTOR =
+    IStreamable(address(AaveV2Ethereum.COLLECTOR));
```

Comment:
```suggestion
```

`IStreamable should no longer be needed. `ICollector` exposes all the methods you need-
https://github.com/bgd-labs/aave-address-book/blob/main/src/common/ICollector.sol#L173